docs/docs/concepts/05-building-blocks/02-events.md

```diff
@@ -0,0 +1,69 @@
+---
+sidebar_position: 2
+---
+
+# Events
+
+Events are discrete actions that change the dynamic state of an AICA application. They are handled and executed by the
+dynamic state engine For this reason, they are sometimes referred to as "state events".
+
+## Event types
+
+The following events can be used to dynamically update the state of a running application:
+
+- Load or unload a component, controller or hardware interface
+- Trigger a lifecycle transition (for example, configure, activate or deactivate) on a lifecycle component
+- Start or stop a controller
+- Set the value of a parameter on a loaded component or controller
+- Call a service on a component
+
+## Triggering events
+
+Events can be triggered externally through interactions with the Developer Interface UI or via the REST API, or
+internally by component predicates and conditions as defined in the application description.
```

Comment:
Here, you refer to predicates just before you explain what they are. Some people might already get worried that they don't know what predicates are. Should me maybe move the following paragraph to the top of the page? Predicates are quite core to the events so having that section up top might not hurt

Reply:
I understand your point but it is really an introduction / transition sentence to the bold section header below. I'll try highlighting / rephrasing it to make it clearer.